src/ctelnet.h

```diff
@@ -75,7 +75,6 @@ const char MXP = 91; // MXP
 const char MSDP = 69; // MSDP, documented at http://tintin.sourceforge.net/msdp/
 
 const char OPT_ECHO = 1;
-const char OPT_SUPPRESS_GA = 3;
```

Comment:
Yep, just a random :thought_balloon: - and here's another: all these `#define`d constants are not the C++ way are they, given that they are (`unsigned`?) `chars` and thus smaller that any `unsigned int` is likely to be (hence ruling out `enum` as those are based on integer types), isn't the _Stroustrup_ way to have `const static char` members with those values; so we'd use things like `ctelnet::csTelnetOptEcho` or just `csTelnetOptEcho` as appropriate?